### DISCUSSION TITLE
[Feature Request] Template Folders

### DISCUSSION THREAD

**Discussion by gigaion** [docusealco/docuseal#44] [Ideas]
Similar to how templates have submissions within them. Templates should be able to be placed inside folders (Maybe even folders nested within another folder also?).
This would allow organizing of content based on category:

Example:
- HR (Folder)
  - Onboarding (Template)
  - Payments (Template)
- NDA (Folder)
  - Top Secret Spaghetti (Template)
  - Spinning Penguin Patrol (Template)
  - Example Nested Folder (Folder)
    - Puppies (Template) 

This would probably tie into/compliment issue #26 , as each template would need the ability to replace PDF documents within a template based on who the document is going out to.

**Comment by AlexBTurchyn** [comment]
@Gigaion I can add a folder system, but no subfolders because that is unnecessary overcomplication

**Comment by omohokcoj** [comment]
@Gigaion folders have been added - only top level folders are available without subfolders
